overlord/fdestate/backend/seal.go

```diff
@@ -47,32 +47,42 @@ var (
 	}
 )
 
-func runKeySealRequests(key secboot.BootstrappedContainer) []secboot.SealKeyRequest {
+func runKeySealRequests(key secboot.BootstrappedContainer, useTokens bool) []secboot.SealKeyRequest {
+	var keyFile string
+	if !useTokens {
```

Comment:
Do we already have a spread test covering this scenario?

Reply:
My guessing is that this is fine because this only done for new seal requests (SealKeys), while ResealKeys checks both tokens and keyfiles (i.e. readKeyData), no?

Reply:
Then it will keep key files. What you had during installation or factory reset is what counts.

Spread tests do tests and will continue to tests cases where we have key files.

Reply:
> My guessing is that this is fine because this only done for new seal requests (SealKeys), while ResealKeys checks both tokens and keyfiles (i.e. readKeyData), no?

Correct.